plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -862,4 +865,99 @@ public function disable( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * When HPOS is enabled, this command lets you remove redundant data from the postmeta table for migrated orders.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <all|id|range>...
+	 * : ID or range of orders to clean up.
+	 *
+	 * [--batch-size=<batch-size>]
+	 * : Number of orders to process per batch. Applies only to cleaning up of 'all' orders.
+	 * ---
+	 * default: 500
+	 * ---
+	 *
+	 * [--force]
+	 * : Whether to skip checks before cleaning up.
```

Comment:
Not completely sure if this is better, but I'm wondering about something like this:

```suggestion
	 * [--force]
	 * : When true, post meta will be cleaned up even if the post appears to have been updated more recently than the order.
```

My line of thinking here is based on the ambiguity of what 'force' means, or of what checks are being skipped. Scenario:

- User runs `wp wc hpos cleanup all`
- Tool reports `Error: Cleanup can only be performed when HPOS is active and compatibility mode is disabled.`
- They want to run the cleanup *anyway* and see that `--force` skips checks, so they try again with `wp wc hpos cleanup all --force`
- Same error, some confusion is experienced and a bug report or support ticket is raised